codice/alliance
# CAL-137 Add email support to NSILI OrderMgr interface

#### What does this PR do?
This PR implements the email destination support found in OrderMgr interface of NSILI.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@troymohl 
@bdeining 
@glenhein 
@emmberk 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith 
@kcwire
#### How should this be tested?
1. Update static variable "emailDest" in NsiliClient.java to a valid email address you can access (make sure "isEmailEnabled" is set to "true")
2. Build Alliance
3. Run instance of Alliance and do a standard full install
4. Upload image through Search UI and click "search" (to add image to catalog)
5. Run command `mvn -Pcorba.client -Dexec.args="https://localhost:8993/services/nsili/ior.txt"` from directory `alliance/distribution/sdk/sample-nsili-client`
6. Verify that no exceptions occur and that an email is sent to your inbox
#### Any background context you want to provide?
Ticket adds support for OrderMgrImpl by implementing the e_dest (email destination) support found in GIAS/Destination.java interface. Previously only http destinations for files were supported.
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Checklist:
